modules/messages/tests/server/message-to-influx.server.service.tests.js

```diff
@@ -0,0 +1,201 @@
+'use strict';
+
+/**
+ * Module dependencies.
+ */
+var mongoose = require('mongoose'),
+    path = require('path'),
+    async = require('async'),
+    User = mongoose.model('User'),
+    Message = mongoose.model('Message'),
+    messageToInfluxService =
+  require(path.resolve('./modules/messages/server/services/message-to-influx.server.service'));
+require('should');
+
+/**
+ * Unit tests
+ */
+describe('Message to influx server service Unit Tests:', function() {
+  var user1,
+      user2;
+
+  // here we create the users
+  beforeEach(function(done) {
+
+    user1 = new User({
+      firstName: 'Full',
+      lastName: 'Name',
+      displayName: 'Full Name',
+      email: 'user1@test.com',
+      username: 'username1',
+      password: 'password123',
+      provider: 'local'
+    });
+
+    user2 = new User({
+      firstName: 'Full',
+      lastName: 'Name',
+      displayName: 'Full Name',
+      email: 'user2@test.com',
+      username: 'username2',
+      password: 'password123',
+      provider: 'local'
+    });
+
+    // save those users to mongoDB
+    user1.save(function(err) {
+      if (err) done(err);
+      user2.save(function(err) {
+        if (err) done(err);
+        done();
+      });
+    });
+  });
+
+  // after each test removing all the messages and users
+  afterEach(function(done) {
+    Message.remove().exec(function() {
+      User.remove().exec(done);
```

Comment:
See https://github.com/Trustroots/trustroots/pull/413#discussion_r80559495

Tests run on a separate db (not in development db) so there's no harm wiping it empty, as it _should_ be empty at the beginning of each new test anyway.